alicja/config/proto/interface.proto

```diff
@@ -0,0 +1,53 @@
+//This proto defines the Hermes - Cloud Prober interface config
+
+syntax = "proto3";
+
+package hermes;
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto"
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+service Hermes_Ceph {
+  //Create and upload a file to the storage system
+  rpc PutFile(PutFileRequest) returns PutFileResponse;
+  
+  //Retrieve a file from the storage system
+  rpc GetFile(GetFileRequest) returns GetFileResponse;
+  
+  //Delete a file from the storage system
+  rpc DeleteFile(DeleteFileRequest) returns DeleteFileResponse;
+}
+
+message HermesFile {
+  string name = 1;
+  targets.TargetsDef target = 2; 
+}
+
+// HermesFile (file name and target) need to be specified so that hermes knows where and what file to create
+message PutFileRequest {
```

Comment:
I think you want to move this comment to the message above 👍 
```suggestion

// HermesFile (file name and target) need to be specified so that Hermes knows what file to create, and which storage system to create it on.
message HermesFile {
  string name = 1;
  targets.TargetsDef target = 2; 
}

message PutFileRequest {
```

Reply:
oh I actually put that there delibaretely but this makes more sense thx!